### ISSUE TITLE
Train on Rectangle Input Sizes

### ISSUE THREAD

**Issue by clydebailey** [Tianxiaomo/pytorch-YOLOv4#535]
I have trained a yolov4-tiny-3l network on custom dataset in alexeyab darknet repo and I am trying to mimic the results using this repo.
The custom model is trained on an image resolution of (320 + 96m)x(320+96n) shape and I see the readme supports this type of rectangular input shape. However In the train.py (yolo loss function), I see only a single argument for image size has been provided of 608. Am I missing anything?

**Comment by pavlik-tikhomirov** [comment]
Train.py is written very bad, so you should change _fsize_ to for example _f_width_ and _f_height_, and then make corresponding changes in init, build_target and forward functions

> I have trained a yolov4-tiny-3l network on custom dataset in alexeyab darknet repo and I am trying to mimic the results using this repo. The custom model is trained on an image resolution of (320 + 96m)x(320+96n) shape and I see the readme supports this type of rectangular input shape. However In the train.py (yolo loss function), I see only a single argument for image size has been provided of 608. Am I missing anything?
